nilearn/_utils/param_validation.py

```diff
@@ -63,3 +63,31 @@ def check_threshold(threshold, data, percentile_func, name='threshold'):
         raise TypeError('%s should be either a number '
                         'or a string finishing with a percent sign' % (name, ))
     return threshold
+
+
+def check_parameters_megatrawls_datasets(inputs, standards, name):
```

Comment:
This doesn't seem like this is general enough to be in _utils.param_validation. Why not put it next to your fetch_megatrawls function.